### PR TITLE
Backend mockbee

### PR DESCRIPTION
**Pull request by Mohitk391**
Mockbee Backend added.

Files to check:
1) ProductContext.js
2) ProductListing.js
3) Login.js
4) Signup.js

preview: https://62d51dc33aae821013e7e0e6--nervous-varahamihira-f6112e.netlify.app/

### CODE REVIEW

**Review by Ankur9669 — APPROVED** [review]
Code looks fine to me

File: src/components/NavBar/NavBar.js [hunk 1]
```diff
@@ -16,8 +16,8 @@ function NavBar() {
             </div>
             <div className="nav-bar-buttons">
                 <Link to="/login"><button className="btn btn-hover sign-in">Login</button></Link>
-                <Link to="/wishlist" className="nav-bar-icon badge-container"><i class="fa fa-heart-o" aria-hidden="true"></i><span class="shopping-badge">{wishlistState.itemsInWishlist}</span></Link>
-                <Link to="/cart" className="nav-bar-icon badge-container"><i class="fa fa-cart-plus" aria-hidden="true"></i><span class="shopping-badge">{items}</span></Link>
+                <Link to="/wishlist" className="nav-bar-icon badge-container"><i className="fa fa-heart-o" aria-hidden="true"></i><span className="shopping-badge">{wishlistState.itemsInWishlist}</span></Link>
+                <Link to="/cart" className="nav-bar-icon badge-container"><i className="fa fa-cart-plus" aria-hidden="true"></i><span className="shopping-badge">{items}</span></Link>
```
Comment (MayankKumar10) [inline src/components/NavBar/NavBar.js:20]: look into indentation here

File: src/backend/controllers/AuthController.js [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+    );
+  } catch (error) {
+    return new Response(
+      500,
+      {},
+      {
+        error,
+      }
+    );
+  }
```
Comment (MayankKumar10) [inline src/backend/controllers/AuthController.js:97]: look into indentation here

File: src/backend/controllers/CartController.js [hunk 3]
```diff
@@ -0,0 +1,148 @@
+import { Response } from "miragejs";
+import { formatDate, requiresAuth } from "../utils/authUtils";
+
+/**
+ * All the routes related to Cart are present here.
```
Comment (MayankKumar10) [inline src/backend/controllers/CartController.js:5]: remove the commented code

**Review by MayankKumar10 — APPROVED** [review]
Look into these Suggestions everything else looks good to me you can merge it.

**Pull request merged**